python/ray/experimental/data/read_api.py

```diff
@@ -12,12 +13,16 @@
 from ray.experimental.data.dataset import Dataset
 from ray.experimental.data.impl.block import ObjectRef, ListBlock, Block
 from ray.experimental.data.impl.arrow_block import ArrowBlock, ArrowRow
+from ray.experimental.data.impl.block_list import BlockList, BlockMetadata
+from ray.experimental.data.impl.lazy_block_list import LazyBlockList
+
+logger = logging.getLogger(__name__)
 
 
 def autoinit_ray(f):
     def wrapped(*a, **kw):
```

Comment:
Nit: Should use `functools.wraps` for at least the `.__name__` and `.__doc__` attrs. https://docs.python.org/3/library/functools.html#functools.wraps